File: src/main/java/no/hvl/tk/visual/debugger/debugging/stackframe/exceptions/StackFrameAnalyzerException.java
package no.hvl.tk.visual.debugger.debugging.stackframe.exceptions;

public class StackFrameAnalyzerException extends RuntimeException {
  public StackFrameAnalyzerException(String message) {
    super(message);
  }

  public StackFrameAnalyzerException(String message, Exception e) {
    super(message, e);
  }
}

File: src/main/java/no/hvl/tk/visual/debugger/debugging/stackframe/exceptions/StackFrameAnalyzerException.kt
package no.hvl.tk.visual.debugger.debugging.stackframe.exceptions;

import com.intellij.debugger.engine.evaluation.EvaluateException;

public class StackFrameAnalyzerException extends RuntimeException {
  public StackFrameAnalyzerException(String message) {
    super(message);
  }

  public StackFrameAnalyzerException(EvaluateException e) {
    super(e);
  }
}
